bitnami/metallb/README.md

```diff
@@ -49,73 +49,82 @@ The command removes all the Kubernetes components associated with the chart and
-| `livenessProbe.successThreshold`                      | Minimum consecutive successes for the probe to be considered successful after having failed.         | `1`                                                     |
-| `livenessProbe.failureThreshold`                      | Minimum consecutive failures for the probe to be considered failed after having succeeded.           | `6`                                                     |
-| `existingConfigMap`                                   | Specify an existing configMapName to use. (this is mutually exclusive with the configInline option)  | `nil`                                                   |
-| `configInline`                                        | Specify the config for metallb as a new configMap inline.                                            | `{}` (does not create configMap)                        |
-| `rbac.create`                                         | Specify if an rbac authorization should be created with the necessarry Rolebindings.                 | `true`                                                  |
-| `prometheus.enabled`                                  | Enable for Prometheus alertmanager basic alerts.                                                     | `false`                                                 |
-| `prometheus.serviceMonitor.enabled`                   | Specify if a servicemonitor will be deployed for prometheus-operator.                                | `true`                                                  |
-| `prometheus.serviceMonitor.jobLabel`                  | Specify the jobLabel to use for the prometheus-operator                                              | `app.kubernetes.io/name"`                               |
-| `prometheus.serviceMonitor.interval`                  | Specify the scrape interval if not specified use defaul prometheus scrapeIntervall                   | `""`                                                    |
-| `prometheus.serviceMonitor.metricRelabelings`         | Specify additional relabeling of metrics.                                                            | `[]`                                                    |
-| `prometheus.serviceMonitor.relabelings`               | Specify general relabeling.                                                                          | `[]`                                                    |
-| `prometheus.serviceMonitor.prometheusRule.enabled`    | Enable prometheus alertmanager basic alerts.                                                         | `true`                                                  |
+| Parameter                                                | Description                                                                                          | Default                                                 |
```

Comment:
There are several parameters missing in the table like those related to podAffinity, xxxxx.podLabels xxxxx.rediness/livenessProbe, `controller.containerPort`, `controller.terminationGracePeriodSeconds`, `psp` and `prometheusRule`.

Also, please divide the table in several smaller ones for readability if you can. Check one of our other [charts](https://github.com/bitnami/charts/tree/master/bitnami/moodle#parameters) for reference.

Reply:
Hmm it is not really satisfying doing that.
Thats why #4142 is now open.
I will see if I can do that somehow.

Reply:
hopefully will have some time tomorrow for getting the documentation right.

Reply:
I am currently working on improving the documentation for the chart.